pkg/resource/poddisruptionbudget.go

```diff
@@ -0,0 +1,88 @@
+package resource
+
+import (
+	"context"
+
+	"github.com/openshift/cluster-image-registry-operator/pkg/defaults"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	policyclient "k8s.io/client-go/kubernetes/typed/policy/v1beta1"
+	policylisters "k8s.io/client-go/listers/policy/v1beta1"
+)
+
+var _ Mutator = &generatorPodDisruptionBudget{}
+
+type generatorPodDisruptionBudget struct {
+	lister policylisters.PodDisruptionBudgetNamespaceLister
+	client policyclient.PolicyV1beta1Interface
+}
+
+func newGeneratorPodDisruptionBudget(lister policylisters.PodDisruptionBudgetNamespaceLister, client policyclient.PolicyV1beta1Interface) *generatorPodDisruptionBudget {
+	return &generatorPodDisruptionBudget{
+		lister: lister,
+		client: client,
+	}
+}
+
+func (gpdb *generatorPodDisruptionBudget) Type() runtime.Object {
+	return &policyv1beta1.PodDisruptionBudget{}
+}
+
+func (gpdb *generatorPodDisruptionBudget) GetNamespace() string {
+	return "openshift-image-registry"
```

Comment:
`defaults.ImageRegistryOperatorNamespace` ?

Reply:
Updated, but it's unclear if we have benefits from this constant.